dlt/common/pipeline.py

```diff
@@ -599,6 +602,28 @@ class StateInjectableContext(ContainerInjectableContext):
         def __init__(self, state: TPipelineState = None) -> None: ...
 
 
+@configspec
+class LoadPackageStateInjectableContext(ContainerInjectableContext):
```

Comment:
cool but why here? it is a part of package storage. and it is a big 👍 that it is decoupled from pipeline (it will help me to handle bad data for example)

Reply:
i moved it, but i am not 100% that it is in the right location now..